src/Razor/Microsoft.NET.Sdk.Razor/src/RazorGenerate.cs

```diff
@@ -125,6 +126,22 @@ protected override string GenerateResponseFileCommands()
                         builder.AppendLine(kind);
```

Comment:
Question not related to this PR but I noticed it while reviewing: @ajaybhargavb / @pranavkm given the `IsNullOrEmpty` check above if someone clears their `DocumentKind` for a Razor item wont that corrupt every other document kind because it will no longer be 1-to-1? I'd imagine if the parser version was >= 3 then we'd always output `Kind` even if it wasn't on the item to ensure the mappings were correct.